app/views/users/edit.html

```diff
@@ -0,0 +1,107 @@
+				<span class="error bg-danger" ng-show="userForm.givenName.$error.maxlength">Max 50 characters</span>
+			</div>
+		</div>
+	</div>
+	<div class="form-group">
+		<div class="row">
+			<div class="col-sm-8">
+				<label for="familyName">Family / Last Name</label>
+				<input type="text" ng-model="user.familyName" class="form-control" name="familyName" placeholder="Last Name" ng-maxlength="50"
+				required>
+			</div>
+			<div role="alert" class="validator col-sm-4">
+				<span class="error bg-danger" ng-show="userForm.familyName.$error.required">Required</span>
+				<span class="error bg-danger" ng-show="userForm.familyName.$error.maxlength">Max 50 characters</span>
+			</div>
+		</div>
+	</div>
+	
+	<h2>Contact Information</h2>
+	<div class="form-group">
+		<div class="row">
+			<div class="col-sm-8">
+				<label for="email">E-mail Address</label>
+				<div class="input-group">
+					<span class="input-group-addon">@</span>
+					<input type="email" ng-model="user.email" class="form-control" name="email" placeholder="E-mail Address" ng-maxlength="256"
+					required>
+				</div>
+			</div>
+			<div role="alert" class="validator col-sm-4">
+				<span class="error bg-danger" ng-show="userForm.email.$error.required">Required</span>
+				<span class="error bg-danger" ng-show="userForm.email.$error.email">Invalid e-mail address</span>
+				<span class="error bg-danger" ng-show="userForm.email.$error.maxlength">Max 256 characters</span>
+			</div>
+		</div>
+	</div>
+	<div class="form-group">
+		<div class="row">
+			<div class="col-sm-8">
+				<label for="primaryPhoneNumber">Primary Phone Number</label>
+				<div class="input-group">
+					<span class="input-group-addon">#</span>
+					<input type="text" ng-model="user.primaryPhoneNumber" class="form-control" name="primaryPhoneNumber" placeholder="Primary Phone Number" ng-pattern="/^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/"
+					required>
+				</div>
+			</div>
+			<div role="alert" class="validator col-sm-4">
+				<span class="error bg-danger" ng-show="userForm.primaryPhoneNumber.$error.required">Required</span>
+				<span class="error bg-danger" ng-show="userForm.primaryPhoneNumber.$error.pattern">Invalid phone number</span>
+			</div>
+		</div>
+	</div>
+	<div class="form-group">
+		<div class="row">
+			<div class="col-sm-8">
+				<label for="secondaryPhoneNumber">Secondary Phone Number</label>
+				<div class="input-group">
+					<span class="input-group-addon">#</span>
+					<input type="text" ng-model="user.secondaryPhoneNumber" class="form-control" name="secondaryPhoneNumber" placeholder="Secondary Phone Number" ng-pattern="/^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$/">
+				</div>
+			</div>
+			<div role="alert" class="validator col-sm-4">
+				<span class="error bg-danger" ng-show="userForm.secondaryPhoneNumber.$error.pattern">Invalid phone number</span>
+			</div>
+		</div>
+	</div>
```

Comment:
I am making careful use of Bootstrap: 

* form-group
* input-group and addons 
* error and bg-danger